packages/cli/src/wrappers/check.ts

```diff
@@ -0,0 +1,25 @@
+import { Arguments } from 'yargs';
```

Comment:
the name of the file is `check.ts` but you are exporting `checkWrapper` function

the name does not match, and I think having it in a seperate file is probalamatic since it feels out of context, should it be next to the file having all the checks ?

Reply:
a wrapper function is wrapping one of the following function types: check, coerce, or middleware.
the wrapper is being passed to a `yargs` `x` function named `check`/`coerce`/`middleware`.
the interface is:
```ts
.check(checkWrapper(specificArgumentCheckFunc, logger))
```
we need to keep a name separation between the wrapping function and the `yargs` function.
than either we keep the name as `{x}Wrapper` or we call it as `x` and import it as `{x}Wrapper`

Reply:
done.